src/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>betrothed</title>
+  </head>
+  <body>
+    <noscript>You need to enable JavaScript to run this app.</noscript>
```

Comment:
This suggestion is dependent on the comment I left on `webpack.config.js`. Just suggestion, please leave as-is if you think that's a better approach!
```suggestion
    <script type="text/javascript" src="main.js"></script>
```

Reply:
Noted in previous comment

Reply:
See my response to your comment on `webpack.config.js`